### ISSUE TITLE
On an iPad, drawer can still be dragged horizontally out of place with a diagonal swipe even with noTouchOpen/Close = true

### ISSUE THREAD

**Issue by fairps** [stoeffel/react-motion-drawer#43]
The content of our drawer extends off the screen on the iPad so the user needs to swipe up and down to scroll around to everything.  If you swipe diagonally, even with noTouchOpen and noTouchClose = true, the drawer drags horizontally and then it stays displaced with stuff slightly off the screen.  If you don't realize how it happened and you continue to swipe up and down, the drawer stays horizontally displaced and you can see part of the content.

Any way to turn off the horizontal movement of the drawer entirely except for the actual open and close?

Thanks!  This is an awesome control and we sure appreciate it!

**Comment by Redmega** [comment]
@fairps Can you verify the `react-motion-drawer` version and `react` versions for me?

**Comment by fairps** [comment]
"react": "^15.6.2",
    "react-motion-drawer": "^2.1.7",

**Comment by fairps** [comment]
Another interesting thing is that the purely horizontal swipe doesn't do anything (as we would expect) but it looks like the diagonal swipe is getting through... I don't know if there is a way to remove the x component of the swipe...

**Comment by Redmega** [comment]
I don't have an iPad to test on, hopefully someone else can help out.

Does the demo have the same problem @fairps? Or is this specifically only when `noTouchClose` and `noTouchOpen` are set to `true`?

**Comment by fairps** [comment]
It was happening with noTouchOpen/Close set false.  We set them true in the hopes that it would fix it.  I just tested the demo page and it happens there as well, but that has noTouchOpen/Close set to true so dragging horizontally or diagonally will close the drawer after it is open.

**Comment by Redmega** [comment]
@fairps Can you upgrade to `v3.0.4` and tell me if this issue still persists?

**Comment by fairps** [comment]
Nope, still there under 3.0.4.  In the example I am trying, I am unable to move the drawer with either a vertical or horizontal drag (it is not large enough to be off the screen) but the diagonal drag still shifts it around.   I don't know if that helps or not.  Thanks for your efforts and let me know if there is anything else I can try for you :-)
